frontend/packages/dev-console/integration-tests/features/topology/topology-display_option.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Display options in topology
+	As a user, I want to see desplay options  
+
+Background:
+    Given user is in topology
+
+@regression, @smoke
+Scenario: Display options menu in topology : T-07-TC10
```

Comment:
Add default options for display menu as one scenario
IMO for each option, the outcomes are different, so I think separate scenarios are helpful